src/com/facebook/buck/swift/SwiftCompile.java

```diff
@@ -151,6 +152,16 @@
                             getProjectFilesystem(), getBuildTarget(), "%s__filelist.txt")))
             : Optional.empty();
 
+    this.argsFilePath =
```

Comment:
Are we setting a unique path per command? If not, could multiple build commands running in parallel possibly overwrite a file that hadn’t been read yet?

(I’m guessing `getBuildTarget()` is what makes this unique, but I don’t know enough to be sure)

Reply:
The similar logic is the *file list path*, which is a few lines above. The arg file path is something like `/path/to/buck-out/bin/ios/lib/Module/Module#apple-swift-compile,iphonesimulator-x86_64__swiftcompile.argsfile`.